_schemaV2.graphql

```diff
@@ -9884,6 +9909,29 @@ type SubmissionEdge {
   node: ConsignmentSubmission
 }
 
+input SubmitInquiryRequestMutationInput {
+  clientMutationId: String
+
+  # The inquireable object id (Artwork ID or Show ID)
+  inquireableID: String!
+
+  # The type of inquireable object (Artwork or Show)
+  inquireableType: String!
+
+  # Optional inquiry message
+  message: String
+
+  # List of structured inquiry questions
+  questions: [InquiryQuestionInput]
```

Comment:
Something we should follow up w/ on both the mutation and the query... we should make these questions non-nullable. What I mean by that is to have the schema represented by `questions: [InquiryQuestionInput]!`. That generally just means we'll _always_ at least return an empty array. If we do that it'll mean less type faffing on the frontend. 

That should be a follow up though, not necessary in this pr.

Reply:
👍 Makes sense. In the back of my mind, I thought this mutation needs to be backward compatible and support the cases where only `message` is present to support force (and I even added tests for that 😄)  but that's not the case.